### PR TITLE
doc: man page. 

### LINKED ISSUE #6: Add a man page

### PR DESCRIPTION
**Pull request by d0p1s4m4**
I open this PR as draft to get early feedback. Because I'm not sure if  you'll agree with 1 man page by sub command (I took inspiration from [cargo(1)](https://manpages.debian.org/testing/cargo/cargo.1.en.html)). 

Issue #6

### CODE REVIEW

**Comment by sleepy-monax** [comment]
I like the direction this is going

**Review by keyboard-slayer — APPROVED** [review]
Looks very clean :+1:
I like it :)

**Comment by d0p1s4m4** [comment]
man pages aren't finished and polished yet but I think it can be merged on dev branch, since it's can be views as an helper to other contributors to write/extend/finish them

File: docs/cutekit-build.1 [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+.SH DESCRIPTION
+
+.SH OPTIONS
+.TP
+\fB\-\-target\fR=\fItarget\fR
+Build target \fItarget\fR.
+
+.SH EXIT STATUS
+exits with status 0.
```
Comment (Supercip971) [inline docs/cutekit-build.1:19]: Maybe add "exits with status 0 on success" ? Not sure. 

:+1:

**Review by Supercip971 — APPROVED** [review]

**Comment by sleepy-monax** [comment]
Closing this one, it's too far behind to be useful

**Pull request closed without merging**